FLEDGE.md

```diff
@@ -784,7 +812,26 @@ When `joinCount` is passed to `generateBid()`, no noising or bucketing is applie
 
 These signals were requested in [issue 435](https://github.com/WICG/turtledove/issues/435). The signals are intented to ship in Chrome M114, they will no longer be available for event level reporting when event level reporting is retired.
 
-#### 5.3 Losing Bidder Reporting
+#### 5.3 Reporting in Multi-Currency Auctions
+
+In auctions that involve multiple currencies, there may be values with different units floating around, which makes aggregated information incomprehensible, and event-level information hard to process, potentially requiring participants to interpret currencies of jurisdictions they do no business in.
+
+To help deal with this scenario, an optional mode is available that converts all bid-related information to seller's preferred currency (in component auctions, reporting for it is for that component's seller). This is configured via the `sellerCurrency` setting in each auction configuration.
+
+Regardless of configuration, the bid passed to `reportWin()` and `reportResult()` is always the bid that participated in that particular level of the auction (which, for top-level auction `reportResult()`, may be the bid altered by the component auction seller's `scoreAd()`), subject to the usual rounding rules. Since passing the returned currency tags around could leak an unacceptable amount of information, the values provided for `browserSignals.bidCurrency` to those methods are the currencies required by the auction configuration (i.e. `sellerCurrency` and `perBuyerCurrencies`), or `'???'` if unspecified.
+
+(Warning: `reportResult` did not follow this rule in Chrome earlier than M116)
+
+All other values, including `highestScoringOtherBid` and winning bids provided to private aggregation and debug event-level loss reporting depend on the `sellerCurrency` setting. We refer to those usages as "potentially mixed currency" contexts, as they can involve information from multiple bids, and hence potentially multiple currencies.
```

Comment:
This seems to contradictory: the first sentence seems to indicate that everything is in `sellerCurrency`, but the second sentence calls this potentially mixed currency and "potentially multiple currencies".
If it would be helpful, feel free to put some time on my calendar to discuss this further and we could work together on this.  I may just not be understanding this very well.

Reply:
What it's trying to convey is that this information originally (potentially) comes from different bids, and therefore it's important to convert it to sellerCurrency.

Reply:
Done, please see https://github.com/morlovich/turtledove/blob/fledge-currency/FLEDGE.md#53-reporting-in-multi-currency-auctions for a preview